public/app/features/scopes/dashboards/ScopesNavigationTreeLink.tsx

```diff
@@ -19,11 +19,13 @@ export function ScopesNavigationTreeLink({ to, title, id }: ScopesNavigationTree
   // For dashboards, the title is appended to the path. We need to diregard this
   const currentPath = isDashboard ? useLocation().pathname.split('/').slice(0, 3).join('/') : useLocation().pathname;
```

Comment:
this feels messy, what do need here really?

Reply:
Dashboard id as such:  `/d/my-dashboard-id`. I agree, it feels a bit fragile.